Client/Frontend/AuthenticationManager/BasePasscodeViewController.swift

```diff
@@ -39,7 +39,11 @@ class BasePasscodeViewController: UIViewController {
         super.viewDidLoad()
         view.backgroundColor = SettingsUX.TableViewHeaderBackgroundColor
         updateRightBarButtonItem()
-        automaticallyAdjustsScrollViewInsets = false
+        if #available(iOS 11.0, *) {
```

Comment:
The minimum iOS version supported for this version will be iOS 12, so this can be removed